asaps/models.py

```diff
@@ -156,15 +154,16 @@ def filter_note_type(client, csv_data, rec_obj, note_type, operation,
 
     Triggers post of updated record if changes are made.
     """
-    for note in rec_obj.updated_json_string['notes']:
+    for note in rec_obj.json_string['notes']:
         try:
             if note['type'] == note_type:
                 for subnote in note['subnotes']:
                     if operation == 'replace_str':
                         fieldval = subnote['content']
```

Comment:
Since we don't need to worry anymore about whether or not a specific field has changed, I think we can just do the replace right here and remove a bunch of code:
```python
subnote['content'] = subnote['content'].replace(old_string, new_string)
```